### PR TITLE
[CIAPP-2379] [DOCS-2294] Document EU1 support

### PR DESCRIPTION
**Pull request by fermayo**
### What does this PR do?

Documents how to use CI App on EU1.

### Motivation

EU1 is now supported by CI App.

### Preview

https://docs-staging.datadoghq.com/fermayo/ciapp-add-eu1/continuous_integration/
https://docs-staging.datadoghq.com/fermayo/ciapp-add-eu1/continuous_integration/setup_tests/agent/
https://docs-staging.datadoghq.com/fermayo/ciapp-add-eu1/continuous_integration/setup_tests/junit_upload/
https://docs-staging.datadoghq.com/fermayo/ciapp-add-eu1/continuous_integration/setup_tests/swift/
https://docs-staging.datadoghq.com/fermayo/ciapp-add-eu1/continuous_integration/setup_pipelines/gitlab/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_pipelines/gitlab.md [hunk 1]
```diff
@@ -108,15 +108,15 @@ As an alternative to using the native Datadog integration, you can use [webhooks
 <div class="alert alert-info"><strong>Note</strong>: The native Datadog integration is the recommended approach and the option that is actively under development.</div>
 
 Go to **Settings > Webhooks** in your repository (or GitLab instance settings), and add a new webhook:
-* **URL**: `https://webhooks-http-intake.logs.datadoghq.com/v2/api/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][5].
+* **URL**: `https://webhooks-http-intake.logs.datadoghq.com/v2/api/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][4].
```
Comment (AdrianLC) [inline content/en/continuous_integration/setup_pipelines/gitlab.md:111]: This also needs a different domain for EU1 in the URL.

**Review by juan-fernandez — APPROVED** [review]

**Review by edrevo — COMMENTED** [review]
There is a "site" menu on the datadog docs website. Maybe we should be using that to show different URLs?

Screenshot 2021-07-26 at 17 12 25

File: content/en/continuous_integration/setup_pipelines/gitlab.md (hunk 1 above)
Comment (fermayo) [reply content/en/continuous_integration/setup_pipelines/gitlab.md:111]: Is it the same just replacing `datadoghq.com` with `datadoghq.eu`?

**Comment by fermayo** [comment]
> There is a "site" menu on the datadog docs website. Maybe we should be using that to show different URLs?
> 
> Screenshot 2021-07-26 at 17 12 25

Good catch. There's a way to change things depending on the selection of site. I'll update the PR with that.

File: content/en/continuous_integration/setup_pipelines/gitlab.md (hunk 1 above)
Comment (AdrianLC) [reply content/en/continuous_integration/setup_pipelines/gitlab.md:111]: yes

**Review by AdrianLC — APPROVED** [review]
Is it possible to restrict the site selector? If you don't read carefully it looks like it would work in all sites, especially now that the selector automatically changes some values.

**Comment by fermayo** [comment]
> Is it possible to restrict the site selector? If you don't read carefully it looks like it would work in all sites, especially now that the selector automatically changes some values.

I've wrapped the webhooks integration to show it only for US1 and EU1

**Review by juan-fernandez — APPROVED** [review]
it looks good, though we show values that are not supported when selecting a site like us3. We show "possible values" so I think that's fine

**Comment by edrevo** [comment]
Screenshot 2021-07-27 at 11 44 50
This is not being rendered correctly

**Comment by fermayo** [comment]
@edrevo fixed it

**Review by edrevo — APPROVED** [review]

**Review by apigirl — APPROVED** [review]

**Pull request merged**